ecommerce_api_client/auth.py

```diff
@@ -5,7 +5,9 @@
 class JwtAuth(AuthBase):
     """Attaches JWT Authentication to the given Request object."""
 
-    def __init__(self, username, full_name, email, signing_key, tracking_context=None):
+    def __init__(self, issuer, expiration, username, full_name, email, signing_key, tracking_context=None):
```

Comment:
`expiration` should be `expires_in` which is the number of seconds after which the token expires. The default value should be 30.